README.md

```diff
@@ -53,3 +54,34 @@ Now, if you go to OpenShift Web Console home page, you'll see the Red Hat Applic
 
 ![screenshot-1](https://user-images.githubusercontent.com/7288588/38807819-273b0f1c-417e-11e8-96d2-c82b41ee59bf.png)
 *screenshot-1: in OpenShift Web Console (v3.7) `Import YAML / JSON` wizard you can paste template raw content*
+## Working with Red Hat Container Development Kit
+If you want to build locally your own images without the need to push them to the Docker repository, you can use Red Hat Container Development Kit (CDK).  
+"Red Hat Container Development Kit provides a pre-built Container Development Environment based on Red Hat Enterprise Linux to help you develop container-based applications quickly." (ref.  [Red Hat Container Development Kit documentation](https://developers.redhat.com/products/cdk/overview/)).  
+For installing CDK, please refer to the https://developers.redhat.com web site where you can find the [Hello World!](https://developers.redhat.com/products/cdk/hello-world/) guide.  
+Once you have a fully working CDK instance, you can follow the next steps:
+
+1. [_optional_]`$ systemctl stop docker` (do this only if you have Docker running on your machine)
+1. `$ minishift docker-env` to display the command you need to type into your shell in order to configure your Docker client since the command output will differ depending on OS and shell type
+1. execute the command from the step before
+1. `$ docker ps` to test that it's working fine and you can see in output a list of running containers
+1. `$ docker login -u developer -p $(oc whoami -t) $(minishift openshift registry)` to log into the CDK OpenShift Docker registry
+1. `$ mvn clean install` to build the Docker images for this project
+1. `$ docker images` to have the list of the images so that you can check to have the built images `windup-web-openshift` and `windup-web-openshift-messaging-executor` and find their TAG value that should be equal the value of the `<version>` tag in the [pom.xml](./pom.xml) file (e.g. `4.1.0-SNAPSHOT`)
```

Comment:
Perhaps we can tell them how to set the tags during the build, instead of having to rename them afterwards every time?

Reply:
Yes, of course, it will be better.